recipes/arcplot/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "arcplot" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arcplot-{{ version }}.tar.gz
+  sha256: 0a85ac7336f8c42f1dfd57b7e0d64b63149f819866456661c0e3408905ca3001
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - poetry-core
+    - pip
+  run:
+    - python >=3.10.0
+    - contourpy 1.2.0
```

Comment:
These pinning are way too strict but I checked and this is exactly what upstream is doing, so let's keep it.